changelogs/fragments/fbsd13_1_remove.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - freebsd 13.1 removed from CI test matrix
```

The changelog entry should cover the `ansible-test` change, rather than the CI change:

```suggestion
  - ansible-test - Removed `freebsd/13.1` remote.
```